package.json

```diff
@@ -50,7 +50,7 @@
     "file-entry-cache": "^2.0.0",
     "functional-red-black-tree": "^1.0.1",
     "glob": "^7.1.2",
-    "globals": "^9.17.0",
+    "globals": "^10.1.0",
```

Comment:
I think this would be a breaking change for ESLint's users, because some globals were removed from the `jest` environment. (See [here](https://github.com/sindresorhus/globals/commit/037a06c8526fa3d9b73e3337d6cd99f530b5dcc5))